src/realm/parser/driver.cpp

```diff
@@ -420,8 +497,21 @@ util::Any LinkAggrNode::visit(ParserDriver* drv)
         case col_type_Decimal:
             sub_column = link_prop->column<Decimal>(col_key).clone();
             break;
-        default:
-            break;
+        case col_type_Bool:
```

Comment:
Better to just use "default". Then we don't have to visit this when we add a new type. If we actually expect it to support aggregation operations, we will get notified.